ingestion/src/metadata/ingestion/source/database/dbt/metadata.py

```diff
@@ -787,23 +812,17 @@ def generate_entity_link(self, dbt_test):
         return entity_link_list
 
     def get_dbt_compiled_query(self, mnode) -> Optional[str]:
-        dbt_query_key_names = ["compiled_sql", "compiled_code"]
-        for key_name in dbt_query_key_names:
-            query = mnode.get(key_name)
-            if query:
-                return query
-        logger.debug(
-            f"Unable to get DBT compiled query for node - {mnode.get('name','unknown')}"
-        )
+        if hasattr(mnode, "compiled_code") and mnode.compiled_code:
```

Comment:
same, these values that have specific and important meaning such as `compiled_code` and `compiled_sql` could be part of an Enum for `COMPILED_QUERIES` or smt.